pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -9941,14 +9921,16 @@ def test_print_object_multi(carma_miriad):
     """
     pytest.importorskip("pyuvdata._miriad")
 
-    _ = carma_miriad._add_phase_center("NOISE", cat_type="unphased", force_update=True)
+    _ = carma_miriad._add_phase_center(
+        "NOISE", cat_type="unprojected", force_update=True
+    )
     check_str = (
-        "   ID     Cat Entry       Type     Az/Lon/RA    El/Lat/Dec  Frame    Epoch \n"
-        "    #          Name                    hours           deg                 \n"
-        "---------------------------------------------------------------------------\n"
-        "    0         NOISE   unphased    0:00:00.00  +90:00:00.00  altaz          \n"
-        "    1         3C273   sidereal   12:29:06.70  + 2:03:08.60    fk5  J2000.0 \n"
-        "    2      1159+292   sidereal   11:59:31.83  +29:14:43.83    fk5  J2000.0 \n"
+        "   ID     Cat Entry          Type     Az/Lon/RA    El/Lat/Dec  Frame    Epoch \n"  # noqa
```

Comment:
Same question on #noqa.

Reply:
Again, it makes the linting pass. I can wrap the lines if you think that's better.